### DISCUSSION TITLE
Scheduled post possible?

### DISCUSSION THREAD

**Discussion by sumberacstefano** [directus/directus#13113] [Archive]
Hi! Is it possible to schedule a post to be 'published' on a specific date in the future. 

For example
Let's say today is 5/4/2022 --> you write down the post with title and description --> you set it to be 'published' on  5/17/2022

**Comment by br41nslug** [comment]
In the current version that is only possible using a custom hook (https://docs.directus.io/extensions/hooks/#schedule).
But once the new [Dataflow Feature](https://github.com/directus/directus/pull/12522) gets released it will be doable without coding!

**Reply by sumberacstefano** [reply]
I will check asap how custom hook is done, thanks!

**Reply by sxsws** [reply]
is there an overview page re data flows? it looks massive and as i'm new to the directus world, i'd like to see a list of features if one exists to help plan several systems i'm designing.

**Reply by benhaynes** [reply]
Nothing formal yet, but you can review the PR (and see some examples) here: https://github.com/directus/directus/pull/12522

**Comment by benhaynes** [comment]
In addition to @br41nslug's suggestion, you can also add a `publish_date` field (any name will do) and then use the API's access control to filter out anything that is not published (eg: `date_publish >= $NOW`). This should work most of the time (with SSR), though for SSG you may need to get creative with when to deploy since this is not a trigger event.

In any event, Data Flows will give far more control over this process.

**Reply by sumberacstefano** [reply]
Thanks @benhaynes `publish_date` field sounds easy
